packages/attackFuncs.js

```diff
@@ -1240,7 +1240,7 @@ function attackEnemy(userName, oppName, userDefs, oppDefs, skillDefs, useEnergy,
 
 			embedText.targetText = `${userDefs.name} => ${oppDefs.name}`
 			embedText.attackText = `${userDefs.name} used ${skillDefs.name}!`
-			embedText.resultText = `${userName} stole ${skillDefs.pow}MP from the target!`
+			embedText.resultText = `${userName} stole **${skillDefs.pow}${userDefs.mpMeter[1]}** from the target!`
```

Comment:
Should this be using the user's MP meter (like it does now) or the foes' MP meter?

Reply:
well, same as the previous one...

Reply:
Indeed.